frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -111,6 +114,10 @@
       if (urlValues.name) form.name = urlValues.name;
       if (urlValues.description) form.description = urlValues.description;
       if (urlValues.type) form.type = urlValues.type;
+      if (urlValues.orgId) form.orgId = urlValues.orgId;
+      if (!form.orgId && myOrgs && myOrgs[0]) {
```

Comment:
this is a little more complicated, but we shouldn't set the org if the Select isn't displayed, so if it's not in devmode, you should be able to import the `isDev` store to check that here.